api/v0/cells/cells.proto

```diff
@@ -167,6 +164,7 @@ message Cell {
 
   CpuController cpu = 2;
   CpusetController cpuset = 3;
+  MemController mem = 4;
```

Comment:
Can we make this `MemoryController` and `memory` to match the controller name as seen in the docs?

Reply:
done